extensions/visualizations/visualizations.js

```diff
@@ -85,12 +88,14 @@ oppia.directive('oppiaVisualizationEnumeratedFrequencyTable', [function() {
       function($scope, $attrs, HtmlEscaperService) {
         $scope.data = HtmlEscaperService.escapedJsonToObj($attrs.data);
         $scope.options = HtmlEscaperService.escapedJsonToObj($attrs.options);
+        $scope.isAddressed = HtmlEscaperService.escapedJsonToObj(
+          $attrs.isAddressed);
+
         $scope.answerVisible = $scope.data.map(function(_, i) {
-          // First element is shown by default, all others are hidden.
-          return i === 0;
+          return i === 0;  // 1st element shown & all others hidden by default.
```

Comment:
Nit: Generally we put comments above the lines they refer to.

Reply:
Done.